SeleniumPracticalExercise/PageObjects/AddEmployeePage.cs

```diff
@@ -19,21 +19,21 @@ public AddEmployeePage(IWebDriver driver) : base(driver)
         /// <summary>
         /// Enter First and Last name
         /// </summary>
-        /// <param name="firstname"></param>
-        /// <param name="lastname"></param>
-        public void SetNames(string firstname, string lastname)
+        /// <param name="firstName">we store firstName.</param>
```

Comment:
>Please< look at the examples in BasePageLocal.cs. This and others need improvement.

Reply:
I looked at BasePageLocal but not sure if I am overthinking it.